src/components/Trailer/Trailer.js

```diff
@@ -0,0 +1,17 @@
+
+export const TrailerMovie = ( videoUrl ) => {
+    const template =  `
+        <video
+        id="my-video"
+        class="video-js vjs-default-skin vjs-big-play-centered"
+        controls
+        preload="auto"
+        width="640"
+        height="268"
+        poster="MY_VIDEO_POSTER.jpg"
```

Comment:
What is that? :o

Reply:
This is poster before turning on the video